### DISCUSSION TITLE
can no longer use `fvm flavor`, has this run option changed?

### DISCUSSION THREAD

**Discussion by alaz-aura** [leoafarias/fvm#637] [Q&A] [answered]
previously  running `fvm flavor x`, it would pick `x` from the `.fvm/fvm_config.json`.

now running `fvm flavor` produces this error:
<img width="489" alt="Screenshot 2024-02-21 at 4 28 24 pm" src="https://github.com/leoafarias/fvm/assets/144073715/aff08472-4480-41c9-aac9-755d6ada33b9">

Could not find a command named "flavor".

**Comment by jibbers42** [comment, marked as answer]
It moved to `use`, see https://github.com/leoafarias/fvm/blob/main/CHANGELOG.md#300

```
> fvm use -h

Sets Flutter SDK Version you would like to use in a project

Usage: fvm use {version}
-h, --help          Print this usage information.
-f, --force         Skips command guards that does Flutter project checks.
-p, --pin           If version provided is a channel. Will pin the latest release of the channel
    --flavor        Sets version for a project flavor
-s, --skip-setup    Skips Flutter setup after install

Run "fvm help" to see global options.
```

**Reply by alaz-aura** [reply]
legend thanks @jibbers42 . I should have checked the changelog, seems the live docs aren't reflecting these changes - https://fvm.app/documentation/guides/basic-commands#flavor

**Reply by leoafarias** [reply]
Thanks @alaz-aura will fix this
